src/vos/tests/evt_ctl.c

```diff
@@ -1440,6 +1448,173 @@ test_evt_iter_delete_internal(void **state)
 	assert_int_equal(rc, 0);
 }
 
+static void
+test_evt_variable_record_size_internal(void **state)
+{
+	struct test_arg		*arg = *state;
+	daos_handle_t		 toh;
+	struct evt_entry_in	 entry = {0};
+	int			 rc, count;
+	int			 epoch;
+	const int		val[] = {D_1K_SIZE,
+					D_16K_SIZE};
+	uint64_t		data_size;
+	char                     *data;
+
+	rc = evt_create(ts_feats, 13, arg->ta_uma, arg->ta_root,
```

Comment:
style: please avoid to use hard coded number as much as possible

Reply:
Sure. I will re-factor the whole C file with a #define EVT_ORDER_DEF_INTERNAL 13

Reply:
Done.